verifiers/src/yang/verifiers/verifiers.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
```

Comment:
Thought the custom verifiers would be in their own custom module and add the class to __init__.py.

__init__.py
```
from .count_verifier import CountVerifier

__all__ = (
    CountVerifier,
    # add custom verifiers here
)
```
If you put all the custom verifiers in one module that module could grow quite large.

Reply:
Yes they will be. Name of the file is confusing i will change it